lib/components/editable_select_component.dart

```diff
@@ -113,6 +113,7 @@ class EditableSelectComponent extends SelectComponent {
   }
 
   get current_input_value => findPart("display_input").value;
+  get name => findPart("input").name;
 
```

Comment:
I suggest this be added into SelectComponent itself, instead of EditableSelectComponent, which inherits from it anyway.